core/PRP/Animation/plLineFollowMod.h

```diff
@@ -41,7 +41,7 @@ class PLASMA_DLL plLineFollowMod : public plMultiModifier {
     };
 
 protected:
-    FollowMode fFollowMode;
+    unsigned short fFollowMode;
```

Comment:
At the time of writing, I didn't really see the value in "strongly" typing this since this is a plain old enum and not an enum class. Wrapping the enum type into Python seemed like an excessive amount of work considering. I can go back and wrap the enum if you prefer, but... :tongue: